en/03_Drawing_a_triangle/03_Drawing/01_Command_buffers.adoc

```diff
@@ -265,26 +234,15 @@ So we need to set them in the command buffer before issuing our draw command:
 
 [,c++]
 ----
-VkViewport viewport{};
-viewport.x = 0.0f;
-viewport.y = 0.0f;
-viewport.width = static_cast<float>(swapChainExtent.width);
-viewport.height = static_cast<float>(swapChainExtent.height);
-viewport.minDepth = 0.0f;
-viewport.maxDepth = 1.0f;
-vkCmdSetViewport(commandBuffer, 0, 1, &viewport);
-
-VkRect2D scissor{};
-scissor.offset = {0, 0};
-scissor.extent = swapChainExtent;
-vkCmdSetScissor(commandBuffer, 0, 1, &scissor);
+commandBuffer->setViewport(0, vk::Viewport(0.0f, 0.0f, static_cast<float>(swapChainExtent.width), static_cast<float>(swapChainExtent.height), 0.0f, 1.0f));
+commandBuffer->setScissor( 0, vk::Rect2D( vk::Offset2D( 0, 0 ), swapChainExtent ) );
 ----
 
 Now we are ready to issue the draw command for the triangle:
 
 [,c++]
 ----
-vkCmdDraw(commandBuffer, 3, 1, 0, 0);
+commandBuffer->draw(3, 1, 0, 0);
 ----
 
 The actual `vkCmdDraw` function is a bit anticlimactic, but it's so simple because of all the information we specified in advance.
```

Comment:
With the tutorial now using slang, the glsl specific built-ins like `gl_vertexIndex` and `gl_InstanceIndex` should be replaced by the Slang variants.